src/main/java/at/porscheinformatik/sonarqube/licensecheck/gradle/GradlePomResolver.java

```diff
@@ -0,0 +1,74 @@
+package at.porscheinformatik.sonarqube.licensecheck.gradle;
+
+import org.apache.commons.io.FileUtils;
+import org.apache.maven.model.Model;
+import org.apache.maven.model.io.xpp3.MavenXpp3Reader;
+import org.codehaus.plexus.util.xml.pull.XmlPullParserException;
+
+import java.io.File;
+import java.io.FileInputStream;
+import java.io.IOException;
+import java.io.InputStream;
+import java.util.Collection;
+import java.util.List;
+import java.util.Objects;
+import java.util.stream.Collectors;
+
+class GradlePomResolver {
+
+    private final File projectRoot;
+
+    GradlePomResolver(File projectRoot) {
+        this.projectRoot = projectRoot;
+    }
+
+    List<Model> resolvePomsOfAllDependencies() throws Exception {
+        File targetDir = resolvePomsAsFiles();
+        return parsePomsInDir(targetDir);
+    }
+
+    private File resolvePomsAsFiles() throws Exception {
+        String relativePoms = "build/poms";
+
+        GradleInvoker gradleInvoker = new GradleInvoker(projectRoot.getAbsolutePath());
+        gradleInvoker.invoke("copyPoms", "-I", createInitScript());
```

Comment:
Does this mean every Gradle project using this plugin has to have a `copyPoms` task?